### DISCUSSION TITLE
At startup: "Configuration Errors" popup with message ":: unknown field"

### DISCUSSION THREAD

**Discussion by cakoose** [ghostty-org/ghostty#3668] [Q&A] [answered]
Ghostty version: 1.0.0, build 8469.
OS: MacOS 14.7.1
Hardware: MacBook Pro M1

1. I just installed Ghostty. I ran it and it seemed to work.
2. I did `Command`+`,` to bring up the config editor and wrote:

```
theme = Builtin Solarized Dark
```

3. When I restarted Ghostty, I got an error:

<img width="952" alt="Screenshot 2024-12-28 at 12 31 16" src="https://github.com/user-attachments/assets/053dc72c-abf6-4e17-8631-9087c8510720" />

I don't see any weird characters in the config:

```
$ hexdump -C ~/.config/ghostty/config
00000000  74 68 65 6d 65 20 3d 20  42 75 69 6c 74 69 6e 20  |theme = Builtin |
00000010  53 6f 6c 61 72 69 7a 65  64 20 44 61 72 6b 0a     |Solarized Dark.|
0000001f
```

Based on the color of the Ghostty terminal window, it looks like it used the `theme` I set.

I searched the Ghostty GitHub discussions for "unknown field": [no results](https://github.com/ghostty-org/ghostty/discussions?discussions_q=%22unknown+field%22).

**Comment by 00-kat** [comment, marked as answer]
Ensure `~/Library/Application Support/com.mitchellh.ghostty/config` doesn't exist, since that is also a supported path for storing configuration.

**Reply by cakoose** [reply]
Oh yup, that was it! For some reason there's a ":" in there.

Would be nice if the Ghostty error message included the file name and line number.

If there's only "\~/Library/Application Support/com.mitchellh.ghostty/config", it shows the file name in the error message. But if there's also a "\~/.config/ghostty/config", it doesn't show the file name.

**Reply by 00-kat** [reply]
> Would be nice if the Ghostty error message included the file name and line number.
> 
> If there's only "\~/Library/Application Support/com.mitchellh.ghostty/config", it shows the file name in the error message. But if there's also a "\~/.config/ghostty/config", it doesn't show the file name.

Consider opening a new discussion about this.

**Reply by naveedunjum** [reply]
> Ensure `~/Library/Application Support/com.mitchellh.ghostty/config` doesn't exist, since that is also a supported path for storing configuration.

Have the same error in Endeavour OS. Any ideas how to solve it?

**Comment by arpangreat** [comment]
also try to see what happens if you put the theme name in quotes, maybe ghostty can't parse whitespaces

**Reply by 00-kat** [reply]
Both no quotes and double quotes work.
